src/create_shards.c

```diff
@@ -245,7 +240,7 @@ master_create_worker_shards(PG_FUNCTION_ARGS)
 		shardStorageType = SHARD_STORAGE_TABLE;
 	}
 
-	for (shardIndex = 0; shardIndex < shardCount; shardIndex++)
+	for (int64 shardIndex = 0; shardIndex < shardCount; shardIndex++)
```

Comment:
I've been declaring loop variables in loops for most of my code. They're loop variables and are bound to the loop (which is the only place they have any meaning). If it were an `if` statement, we'd push their definition inside the braces: the only reason we don't for loops is because others don't consider it because it's C99.

I like it, and `cppcheck` gripes at me if I don't do it (since scope of the variable could be reduced).